### PR TITLE
PP-642/migrate-test-utils-server

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

- Migrate utility files used by the server

## Why

- Necessary to integration test

## Refs

- [PP-642](https://rsklabs.atlassian.net/browse/PP-642)

### CODE REVIEW

File: src/AccountManager.ts [hunk 1]
```diff
@@ -11,8 +11,18 @@ import {
 } from './typedRequestData.utils';
 
 export default class AccountManager {
+  private static instance: AccountManager;
+
   private _accounts: Wallet[] = [];
 
+  public static getInstance(): AccountManager {
```
Comment (antomor) [inline src/AccountManager.ts:18]: I think we should create a private constructor to avoid the creation of multiple instances

**Review by antomor — CHANGES_REQUESTED** [review]
@franciscotobar We need to slightly change `accountManager` to implement the singleton

File: src/AccountManager.ts (hunk 1 above)
Comment (franciscotobar) [reply src/AccountManager.ts:22]: updated

**Review by antomor — APPROVED** [review]
LG

**Pull request merged**